hs_core/templates/pages/baseresource.html

```diff
@@ -699,7 +699,7 @@ <h4 class="modal-title">Confirm files deletion</h4>
                                         <div id="fb-file-operations-controls" class="col-xs-12 col-sm-12">
                                             {% if page.perms.change and resource_edit_mode and cm.get_supported_upload_file_types != "()" %}
                                                 {% if not cm.raccess.published_or_review_pending or current_user.is_superuser %}
-                                                    <a {% if remaining_quota <= 0 %}disabled{% endif %} id="uppy-modal-trigger" type="button" data-fb-action="uploadFiles"
+                                                    <a {% if remaining_quota <= 0 %}disabled{% else %} id="uppy-modal-trigger" {% endif %} type="button" data-fb-action="uploadFiles"
```

Comment:
@devincowan - I noticed that even thought the Add files button is disabled, the uppy-modal opens when you click the button. If you have a more elegant solution,  please change this.

Reply:
nice catch!